uxarray/constants.py

```diff
@@ -0,0 +1,5 @@
+import numpy as np
+
+# is 32-bit safe or should we use 64-bit?
+INT_DTYPE = np.uint32
+FILL_VALUE = np.iinfo(INT_DTYPE).max
```

Comment:
Should this be INTFILL_VALUE ?   What are doing for data on the grid that is in float and comes with a _FillValue?  Do we still want those to be Nan?

Reply:
In https://github.com/UXARRAY/uxarray/issues/189#issuecomment-1428509976 you thought we'd need FILL_VALUE32 and FILL_VALUE64.

Reply:
Good point, we should specify that this fill value is for integer arrays, I like the name `INT_FILL_VALUE`

The `replace_fill_values()` method handles converting data types and pre-defined fill values to the standardized versions, with correct edge case handling.

I think `FLOAT_FILL_VALUE` should be set to `NaN`, but this PR is only standardizing the connectivity variables, which should never be floats. Once we get these changes merged we can standardize the float variables to use a consistent fill value.

Reply:
> In [#189 (comment)](https://github.com/UXARRAY/uxarray/issues/189#issuecomment-1428509976) you thought we'd need FILL_VALUE32 and FILL_VALUE64.

After giving it some more thought, for connectivity variables I don't see a situation where we'd need more than 32-bit precision. However, we could use [`np.uintp`](https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.intp:~:text=%27p%27-,numpy.uintp,%23,-Alias%20for%20the) which would dynamically change to either 32 or 64 bit depending on the system.